### DISCUSSION TITLE
Command Install HyperHDR to Raspberry PI

### DISCUSSION THREAD

**Discussion by GaryEl** [awawa-dev/HyperHDR#314] [Q&A]
Hi can you write me the command for install directly on SSH to my Raspberry Pi 3 ? Thanks

**Comment by awawa-dev** [comment]
https://github.com/awawa-dev/HyperHDR/wiki/Installation#raspberrypi-os-sd-card-image-installation

**Comment by GaryEl** [comment]
I try to install sudo apt install HyperHDR-17.0.0.0-Linux-armv7l.deb but this is not work, I want install like all package, Hyperion, Homberidge, PiHole etc
